taca_ngi_pipeline/cli.py

```diff
@@ -139,9 +179,12 @@ def sample(ctx, projectid, sampleid):
                 projectid,
                 sid,
                 **ctx.parent.params)
-        elif ctx.parent.params['cluster'] == 'grus': # Hard stage and deliver (not implemented, use project)
+        elif ctx.parent.params['cluster'] == 'grus':
             logger.error("When delivering to grus only project can be specified, not sample")
             return 1
+        elif ctx.parent.params['cluster'] == 'dds':
+            logger.error("When delivering with DDS only project can be specified, not sample")
```

Comment:
Which project should be specified, the DDS project?

Reply:
No. This is a separate option `taca deliver sample`, which would allow you to deliver just a sample instead of a whole project. But it's not implemented for grus, and we probably don't want it for dds either, so I'm thinking about removing it completely.